src/transaction.js

```diff
@@ -643,6 +643,32 @@ const transaction = {
     }
   },
 
+  /**
+   * Generate a transaction and returns it, without propagating to the network.
+   **/
```

Comment:
Still missing docstring about parameters, return and memberof

Reply:
Fixed. Thanks!